<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">
*
{
font: 75 14pt &quot;Calibri&quot;;
}
QLineEdit
{
font: 12pt &quot;Calibri&quot;;
}
QComboBox
{
font: 12pt &quot;Calibri&quot;;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Add / Visit</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="title">
            <string>Add</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="nameLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>Username</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="passLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QComboBox" name="roleComboBox">
                  <property name="minimumSize">
                   <size>
                    <width>185</width>
                    <height>32</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>Admin</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>HR</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Sales</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Workshop</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Parts Store</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="addPushButton">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton:hover
{
background-color: rgb(0, 85, 255);

}

QPushButton 
{
     font: 75  14pt &quot;Calibri&quot;;
	background-color: rgb(0, 170, 255);
	
	color: rgb(255, 255, 255);

	border-radius: 5px;
}</string>
                  </property>
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="title">
            <string>Visit</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Department</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>30</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>Human Resources</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Workshop</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Sales</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Parts Store</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Parts Processing</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="okPushButton">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>32</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton:hover
{
background-color: rgb(0, 85, 255);

}

QPushButton 
{
     font: 75  14pt &quot;Calibri&quot;;
	background-color: #1FDA9A;
	
	color: rgb(255, 255, 255);

	border-radius: 5px;

}</string>
                  </property>
                  <property name="text">
                   <string>Visit</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Edit</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="removePushButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:disabled {
	
	background-color: rgb(198, 198, 198);
}
QPushButton:hover
{
background-color: rgb(182, 0, 0);

}

QPushButton 
{
     font: 75  14pt &quot;Calibri&quot;;
	background-color: rgbrgb(255, 0, 0);
	
	color: rgb(255, 255, 255);

	border-radius: 5px;
}</string>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
